braindecode/models/eegitnet.py

```diff
@@ -0,0 +1,313 @@
+                nn.BatchNorm2d(14),
+                nn.ELU(),
+                nn.Dropout(drop_prob),
+            ]
+        )
+
+        self.block5 = nn.Sequential(
+            *[
+                DepthwiseConv2d(
+                    14,
+                    kernel_size=(1, 4),
+                    depth_multiplier=1,
+                    dilation=(1, 2),
+                    bias=False,
+                    padding="valid",
+                ),
+                nn.BatchNorm2d(14),
+                nn.ELU(),
+                nn.Dropout(drop_prob),
+            ]
+        )
+
+        # ================================
+
+        self.block6 = nn.Sequential(
+            *[
+                DepthwiseConv2d(
+                    14,
+                    kernel_size=(1, 4),
+                    depth_multiplier=1,
+                    dilation=(1, 4),
+                    bias=False,
+                    padding="valid",
+                ),
+                nn.BatchNorm2d(14),
+                nn.ELU(),
+                nn.Dropout(drop_prob),
+            ]
+        )
+
+        self.block7 = nn.Sequential(
+            *[
+                DepthwiseConv2d(
+                    14,
+                    kernel_size=(1, 4),
+                    depth_multiplier=1,
+                    dilation=(1, 4),
+                    bias=False,
+                    padding="valid",
+                ),
+                nn.BatchNorm2d(14),
+                nn.ELU(),
+                nn.Dropout(drop_prob),
+            ]
+        )
+
+        # ================================
+
+        self.block8 = nn.Sequential(
+            *[
+                DepthwiseConv2d(
+                    14,
+                    kernel_size=(1, 4),
+                    depth_multiplier=1,
+                    dilation=(1, 8),
+                    bias=False,
+                    padding="valid",
+                ),
+                nn.BatchNorm2d(14),
+                nn.ELU(),
+                nn.Dropout(drop_prob),
+            ]
+        )
+
+        self.block9 = nn.Sequential(
+            *[
+                DepthwiseConv2d(
+                    14,
+                    kernel_size=(1, 4),
+                    depth_multiplier=1,
+                    dilation=(1, 8),
+                    bias=False,
+                    padding="valid",
+                ),
+                nn.BatchNorm2d(14),
+                nn.ELU(),
+                nn.Dropout(drop_prob),
+            ]
+        )
+
+        # ================================
+
+        self.block_reduce = nn.Sequential(
+            *[
+                nn.Conv2d(14, 28, kernel_size=(1, 1)),
+                nn.BatchNorm2d(28),
+                nn.ELU(),
+                nn.AvgPool2d((1, 4)),
+                nn.Dropout(drop_prob),
+            ]
+        )
+
+        self.classifier = nn.Linear(
+            int(int(input_window_samples / 4) / 4) * 28, n_classes
+        )
+        self.m = nn.Softmax(dim=1)
```

Comment:
In the case of `nn.Sequential` we can just use `self.add_module` method to add all the components to the model

Reply:
done